RFC-0004-sparse-fill-value/README.md

```diff
@@ -0,0 +1,613 @@
+# PyTorch Sparse Tensors: `fill_value` property
```

Comment:
One can think of titles more descriptive to people not familiar with the topic at hand.
```suggestion
# Expanding PyTorch sparse tensor support with fill values
```

Reply:
Thanks for the suggestion. Perhaps: "Extending PyTorch sparse tensors with fill value" to make it more concrete.

Reply:
Or: "Adding fill value property to PyTorch sparse tensors"